src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -645,11 +637,15 @@ function MultiChainPositions({
   const positions = value?.positions;
   const items = useMemo(
     () =>
-      chainValue === NetworkSelectValue.All || !positions
-        ? positions
-        : positions.filter((position) => position.chain === chainValue),
+      positions?.filter(
+        (position) =>
+          (chainValue === NetworkSelectValue.All ||
+            position.chain === chainValue) &&
+          (position.type === 'asset' ? position.is_displayable : true)
```

Comment:
move this condition first so that we compare booleans before comparing strings :)